core/src/main/java/org/pentaho/di/connections/vfs/VFSConnectionDetails.java

```diff
@@ -38,10 +38,13 @@ default String getDomain() {
     return "";
   }
 
+  default boolean hasBuckets() {
```

Comment:
Moved from `ConnectionDetails`, as this is VFS-specific.

Reply:
See corresponding discussion in `ConnectionDetails`: https://github.com/pentaho/pentaho-kettle/pull/9334/files#r1609796559.

Reply:
In the meantime, this was addressed in another PR.